tensorflow_addons/rnn/BUILD

```diff
@@ -7,6 +7,7 @@ py_library(
     srcs = [
         "__init__.py",
         "cell.py",
+        "layernorm_simplernn.py",
```

Comment:
`layernorm_simplernn.py` has over 400 lines of code, and has 3 additional lines for imports (e.g. `SimpleRNN` instead of `tf.keras.layers.SimpleRNN`, or `K` instead of `tf.keras.backend`). It's what used in tensorflow/keras' `recurrent.py`. Should `K.dot` replaced with `tf.keras.backend.dot` in `call()`?

Reply:
moved the code to cell.py https://github.com/tensorflow/addons/pull/841/commits/6fff49fb4ee4151a4c34f364f7610283ec841a51